<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manageUserForm</class>
 <widget class="QWidget" name="manageUserForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="UserLists">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>251</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>用户列表：</string>
   </property>
   <property name="buddy">
    <cstring>UserLists</cstring>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>181</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>用户名：</string>
        </property>
        <property name="buddy">
         <cstring>EditUserName</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="EditUserName"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>输入密码：</string>
        </property>
        <property name="buddy">
         <cstring>EditPasswd</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="EditPasswd">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>确认密码：</string>
        </property>
        <property name="buddy">
         <cstring>ConfirmPasswd</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ConfirmPasswd">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="deleteUserButton">
        <property name="text">
         <string>删除用户</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addUserButton">
        <property name="text">
         <string>添加用户</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="exitButton">
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>EditUserName</tabstop>
  <tabstop>EditPasswd</tabstop>
  <tabstop>ConfirmPasswd</tabstop>
  <tabstop>deleteUserButton</tabstop>
  <tabstop>addUserButton</tabstop>
  <tabstop>exitButton</tabstop>
  <tabstop>UserLists</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
